build-apisix-openresty-asan-centos7.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x
+
+yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
+yum install centos-release-scl -y
+yum -y install gcc gcc-c++ patch wget git make sudo libasan
+yum -y install openresty-openssl111-asan openresty-pcre-asan openresty-zlib-asan
+
+export openssl_prefix=/usr/local/openresty/openssl111
+export zlib_prefix=/usr/local/openresty/zlib
+export pcre_prefix=/usr/local/openresty/pcre
+
+export ASAN_OPTIONS=detect_leaks=0
+export OR_PREFIX="/usr/local/openresty-asan"
+export cc_opt="-DNGX_LUA_ABORT_AT_PANIC -I${zlib_prefix}/include -I${pcre_prefix}/include -I${openssl_prefix}/include -O1"
+export ld_opt="-L${zlib_prefix}/lib -L${pcre_prefix}/lib -L${openssl_prefix}/lib -Wl,-rpath,${zlib_prefix}/lib:${pcre_prefix}/lib:${openssl_prefix}/lib"
+export cc='--with-cc="ccache gcc -fsanitize=address"'
```

Comment:
The official openresty-asan uses clang to build itself

Reply:
I saw this https://github.com/openresty/openresty-packaging/commit/d7fb0a7814151033aa0c5babbb9fd40e48f49905. 
and I'm building `openresty-asan` on an arm64 machine and it's confusing me. It always prompts me with
 `Error: Package: openresty-openssl111-asan-1.1.1k-1.el7.aarch64 (openresty)
           Requires: libasan
`
later I will try to build `openresty-asan` on an x86_64 machine